src/main/java/iohomework/Main.java

```diff
@@ -0,0 +1,78 @@
+package iohomework;
+
+import com.github.javafaker.Faker;
+import org.apache.poi.hssf.usermodel.*;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Main {
+    static final String localDir = System.getProperty("user.dir");
+
+    public static void main(String[] args) {
+        createExampleCsv();
+        writeToExcelFile(readFromCsvFile());
+    }
+
+    public static void createExampleCsv(){
+        Faker fakeData = new Faker();
+        try {
+            FileWriter exampleCSV = new FileWriter(localDir + File.separator
+                    + "src" + File.separator + "main" + File.separator
+                    + "resources" + File.separator +"source.csv");
+            String dataString = IntStream.range(0, 10)
+                    .mapToObj(i -> fakeData.name().firstName())
+                    .collect(Collectors.joining(","));
+            exampleCSV.write(dataString);
+            exampleCSV.flush();
+            exampleCSV.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static List<String> readFromCsvFile(){
+        List<String> extractedData = new ArrayList<>();
+        try{
+            FileReader dataSource = new FileReader(localDir + File.separator
+                    + "src" + File.separator + "main" + File.separator
+                    + "resources" + File.separator +"source.csv");
+            BufferedReader reader = new BufferedReader(dataSource);
+            while(true){
```

Comment:
Overall good code but here is a bad practice - while without exit condition 
Standart approach is
```
BufferedReader r = new BufferedReader(new FileReader(myFile));
String s = r.readLine();
while (s != null) {
    // do something with s
    s = r.readLine();
}
r.close();
```

Reply:
change while condition